sparkplug-executor/src/main/scala/springnz/sparkplug/executor/ExecutorService.scala

```diff
@@ -10,25 +11,41 @@ import springnz.sparkplug.util.Logging
 import scala.util.{ Properties, Try }
 
 object Constants {
-  val defaultConfigSectionName = "sparkPlugAkkaExecutorService"
-  val actorSystemName = "sparkPlugExecutorSystem"
-  val brokerActorName = "sparkPlugRequestBroker"
+  val defaultAkkaRemoteConfigSection = "akkaRemote"
+  val actorSystemName = "sparkplugExecutorSystem"
+  val brokerActorName = "sparkplugRequestBroker"
 }
 
 object ExecutorService extends Logging {
   import Constants._
 
+  lazy val defaultRemoteAkkaConfig = ConfigFactory.load.getConfig(s"sparkplug.$defaultAkkaRemoteConfigSection")
+
   // TODO: proper command line parsing to allow richer config options
   def main(args: Array[String]): Unit = {
-    if (args.length != 2)
-      throw new IllegalArgumentException(s"Expected 2 arguments to ExecutorService, the app name and the akka address of the client actor. Args = : ${args.toList}")
-    val appName = args(0)
-    val sparkClientPath = args(1)
+    if (args.length < 4)
+      throw new IllegalArgumentException(s"Expected at least 4 arguments to ExecutorService, the app name and the akka address of the client actor. Args = : ${args.toList}")
```

Comment:
Error message no longer makes sense, as it only mentions two of the required arguments.